### ISSUE TITLE
If I were to use pretrained efficientnet for only inference?

### ISSUE THREAD

**Issue by ooodragon94** [lukemelas/EfficientNet-PyTorch#182]
https://github.com/lukemelas/EfficientNet-PyTorch/issues/92#issuecomment-552133128
This tells us that we need to upscale the imagenet's resolution to get specified accuracy
and when I see the code it seems like it's only done when doing validation

I'm trying to distill efficientnetb5 to b0 (both pretrained)

How should I go and change the code?
should b5 inferenced on specified upscaled image and b0 with 224 by 224 resolution??

plz help! thank you
